app/application.js

```diff
@@ -213,7 +172,19 @@ export class DesktopApplication {
     });
 
     winState.manage(this.window);
-    this.onWindowCreate();
```

Comment:
Hmm, I agree that this function was not necessary. But now the createWindow func is now doing more than creating the window—it's also creating a bunch of services. We could either rename this to `createWindowAndServices` or separate the two.